react/react1/week2/app/src/components/todosData.jsx

```diff
@@ -0,0 +1,16 @@
+const todos = [
+  {
+    id: 1,
+    description: "Get out of bed",
+  },
+  {
+    id: 2,
+    description: "Brush teeth",
+  },
+  {
+    id: 3,
+    description: "Eat breakfast",
+  },
+];
```

Comment:
Nitpick, but this would not hold any JSX, so it could (should?) be just a .js file 👍🏽

Reply:
Hi Karo,
thanks for reviewing my hw.
Regarding the .jsx file, I thought we should always use it in React.
I'll read about .js vs. .jsx to understand the concepts behind them. 🙂

Reply:
Essentially the files are completely interchangeable, as far as I know. 
It is not that it is super important here, that is why I said it is a nitpick 😅 
It is just clearer if you keep .jsx for components and .js for helpers and data files, but it is not a convention, more like what I have seen, but you could choose your preference 🙏🏽 
I have just through some replies in this https://stackoverflow.com/questions/46169472/reactjs-js-vs-jsx, and it provides some insight.

Reply:
Yes, anyway it was good to read about it and clarify some doubts.
Thanks. 👍🏻